README.md

```diff
@@ -90,8 +90,7 @@ Have a question, bug, or feature request? Let us know! https://kind.sigs.k8s.io/
 Control-plane nodes need to remove the special label `node.kubernetes.io/exclude-from-external-load-balancers` to be able to access the workloads running on those nodes using a LoadBalancer Service.
 
 ```sh
-$ kubectl label node kind-control-plane node.kubernetes.io/exclude-from-external-load-balancers-
-node/kind-control-plane unlabeled
+kubectl label node kind-control-plane node.kubernetes.io/exclude-from-external-load-balancers=true
```

Comment:
```suggestion
kubectl label node kind-control-plane node.kubernetes.io/exclude-from-external-load-balancers-
```

the goal is to remove the label not to set it to true, no?

Reply:
yes, it seems you are right that was the opposite of what I understood because it was unlabelled when I got a failed command but I think you are right. It either was never labelled or (more likely) I managed to unlabel it without realizing.